schedule.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      W3C Workshop on Web &amp; Virtual Reality
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <p>
+      <a href="https://www.w3.org/"><img alt="W3C" src=
+      "https://www.w3.org/Icons/w3c_home" height="48" width="72"></a>
+    </p>
+    <h1 id="top">
+      W3C Workshop on Web &amp; Virtual Reality
+    </h1>
+    <p>
+      October 19-20, 2016; San Jose, CA, USA
+    </p>
+    <ul class="menu">
+      <li>
+        <a href="#goals">Goals</a>
+      </li>
+      <li>
+        <a href="#attend">How to Attend</a>
+      </li>
+      <li>
+        <a href="#location">Location</a>
+      </li>
+      <li>
+        <a href="#program">Program Committee</a>
+      </li>
+      <li>
+        <a href="papers.html">Position Statements</a>
+      </li>
+      <li>
+        <a class="active-tab">Workshop Schedule</a>
+      </li>
+    </ul>
+    <aside class="box" id="host">
+      <h2 class="footnote">
+        Host
+      </h2>
+      <p>
+        W3C gratefully acknowledges Samsung for hosting this workshop.
+      </p>
+      <p>
+        <a href="http://samsung.com/"><img src="samsung.svg" alt="Samsung"></a>
+      </p>
+    </aside>
+    <aside class="box" id="sponsor">
+      <h2 class="footnote">
+        Sponsors
+      </h2>
+      <article class="sponsors">
+        <h3 id="platinum-sidebar">
+          Platinum Sponsor
+        </h3>
+        <p>
+          <a href="https://www.google.com/"><img src="google.svg" alt=
+          "Google"></a><br>
+          <a href="https://www.mozilla.org/"><img src="mozilla.svg" alt=
+          "Mozilla"></a>
+        </p>
+      </article>
+      <p>
+        <a href="./#sponsorship">Become a sponsor!</a>
+      </p>
+      <p>
+        <a href="sponsorship.html">Sponsorship package info.</a>
+      </p>
+    </aside>
+    <main>
+      <h3 id="schedule">
+        Tentative Schedule
+      </h3>
+      <p>
+        The workshop will focus around several topics identified by the
+        expressions of interest. Each topic will be introduced by one or more
+        related lightning talks, and will be explored more in-depth by
+        discussion breakouts, concluded with joint summaries of the breakouts.
+        The goal of the discussion is not be to resolve the technical issues of
+        the topic, but to determine its relevance and priority to
+        standardization.
+      </p>
+      <p>
+        <b>This schedule may change based on discussions with the program
+        committee.</b>
+      </p>
+      <h4 id="day1">
+        Day 1: 19 October
```

Comment:
I know it's European format, but to be consistent with the usage elsewhere, can you change to this?

```
Day 1: Wednesday, October 19, 2016
```